source/install_compass.txt

```diff
@@ -0,0 +1,100 @@
+.. _install-compass:
```

Comment:
Same thought w.r.t. Compass - its not a commandline tool. If anything, after we finish migration we would leave this as part of the MongoDB server installation side of things.